src/ser.rs

```diff
@@ -22,6 +22,31 @@ impl Serialize for std::time::Duration {
     }
 }
 
+impl Serialize for Option<std::time::Duration> {
```

Comment:
Probably a dump question, but why support both your own time type and `std::time:Duration` for `Serialize`?

Reply:
AFAIK serde didn't have serialization of any `Duration` to nanoseconds, neither `chrono` nor `std::time` so it's nice to support more use cases 🙂 .

Reply:
Feature gated support for the chrono crate, it's a different type with a bit different representation and precision.